odata-mock/package.json

```diff
@@ -38,7 +38,7 @@
     "copyfiles": "^2.1.0",
     "mocha": "^6.1.3",
     "supertest": "^4.0.2",
-    "typescript": "~3.4.3"
+    "typescript": "~3.2.2"
```

Comment:
please use ^ like for all other dependencies

Reply:
doesn't that mean to use a higher version if available?

Reply:
yes, compatible minor version, not major version

Reply:
but I need it to be exactly that because angular doesn't support higher versions...and lerna wants versions to be the same across projects

Reply:
ok, then keep it